rescript/plugin_setup.py

```diff
@@ -483,20 +485,44 @@
         'replacement_map': 'A tab-delimitad metadata file in which '
                            'the rank-lineage string in the \'id\' '
                            'column is replaced by the rank-lineage '
```

Comment:
"column is replaced by the rank-lineage string. All string in a second named column."

Maybe change to:
"column is replaced by the replacement string(s) listed in the second column."